dist/src/assembly/bin/zeneventserver-restart

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+. ~/.bashrc
+. $ZENHOME/bin/zenfunctions
+
+LOG=$ZENHOME/log/zep-restart.log
+
+echo "==========================================================" >> $LOG
+echo "`date` Restarting zep because of OutOfMemoryError " >> $LOG
+
+#Setting ZENOSS_STARTUP_TIMEOUT = 0 to kill zeneventserver process
+export ZENOSS_STARTUP_TIMEOUT=0
+zeneventserver stop >> $LOG 2>&1
+#Unsetting ZENOSS_STARTUP_TIMEOUT to default
+unset ZENOSS_STARTUP_TIMEOUT
+### Starting zeneventserver 
+zeneventserver start >> $LOG 2>&1
```

Comment:
Should not try to start processes inside a container. The serviced container-controller will restart the process if it dies, stops or is killed.